Replace link to wiki with link to archive on WayBack Machine

cstrobbe opened the pull request:
The wiki at wiki.gpii.net is no longer online, so the link points to nowhere.

cstrobbe committed:
Remove link to wiki

The wiki at wiki.gpii.net is no longer online, so the link points to nowhere.

amb26 commented:
@duhrer, @gtirloni - Do I remember that someone has a dump of the wiki somewhere? We should refresh our memories about where it is, even if we don't have the resources to put it online ...

colinbdclark commented:
It's [archived in the Wayback Machine](https://web.archive.org/web/20200807234452/http://wiki.gpii.net/w/Architecture), which seems a more reasonable place to link to as anything we'd host ourselves.

amb26 commented:
> It's [archived in the Wayback Machine](https://web.archive.org/web/20200807234452/http://wiki.gpii.net/w/Architecture), which seems a more reasonable place to link to as anything we'd host ourselves.

Great idea - shall we update the link to that?

cstrobbe committed:
Use link to Archive.org

Replace text deleted in previous commit with link to version of wiki on Wayback Machine.
